test/test-functions.json

```diff
@@ -1,4 +1,85 @@
 {
+  "date": [
+    { "src": "{:date}", "exp": "{:date}", "errors": [{ "type": "bad-input" }] },
+    {
+      "src": "{horse :date}",
+      "exp": "{|horse|}",
+      "errors": [{ "name": "RangeError" }]
```

Comment:
I believe the spec says this should be an invalid expression error. Your code might call this a `RangeError`. Fix?

Reply:
Ah, true. I currently don't add a wrapper for the built in JS RangeError that's the cause here, as the `:number` spec requires.